lib/curator/serializers/schema.rb

```diff
@@ -159,6 +158,28 @@ def with_cache(record)
 
       private
 
+      # def parse_val(val)
```

Comment:
we possibly may need this in the future to parse values that are hashes or arrays in order to recursively check if the values within are nil or empty. It may not be needed once during the refactor of the serializer. But I can add a comment on this in the meantime describing why it may be needed